### PR TITLE
Update firmware guide

### LINKED ISSUE #507: [Typo]: Redundant tip on mac network folder guide
### Prerequisites

- [X] I have searched for existing issues that already report this problem.

### Device

Laptop/Desktop

### Device OS

MacOS

### Device OS Version

Ventura

### Browser

Firefox

### Browser Version

117

### Current Behavior

Simple typo: the tip located at step 8 of the guide is redundant. The same tip is mentioned twice.

### Expected Behavior

only one needed

### Steps to Reproduce

go to https://docs.start9.com/latest/guides/device-guides/dg-mac/backup-mac

### Anything else?

BEE969F8-401E-478E-9259-40BC81F40DC4

### PR DESCRIPTION
**Pull request by kn0wmad**
Also closes #507

### CODE REVIEW

File: site/source/guides/flashing/flashing-firmware.rst [hunk 1]
```diff
@@ -3,24 +3,19 @@
 ===================
 Flashing (Firmware)
 ===================
-This page is for existing Librem Mini owners ONLY.  This will add StartOS-specific tweaks to your system, such that it will operate just as a Start9 Pure bought from the `Start9 Store <https://store.start9.com/collections/servers/products/server-pure>`_.
+This page is for Server Pro / Purism Librem Mini users ONLY.  This firmware is custom for these devices.
```
Comment (gStart9) [inline site/source/guides/flashing/flashing-firmware.rst:6]:
```suggestion
This page is for Server Pure / Purism Librem Mini users ONLY.  This firmware is custom for these devices.
```

File: site/source/guides/flashing/flashing-firmware.rst [hunk 2]
```diff
@@ -3,24 +3,19 @@
...
+The source code can be viewed on Purism's `firmware git repo <https://source.puri.sm/firmware/releases/-/tree/master/librem_mini_v2/custom>`_.  You will need a USB flash drive, formatted FAT32, to flash the firmware to your server.
 
-#. `Download the firmware from Purism's git repository <https://source.puri.sm/firmware/releases/-/raw/master/librem_mini_v2/custom/pureboot-librem_mini_v2-basic_usb_autoboot_blob_jail-Release-27.1.rom.gz>`_ and extract it.  You'll need a USB flash drive, formatted FAT32.
-    
-    .. warning:: USB drive **must** be formatted as FAT32
-
-#. Copy or move the resulting ``pureboot-librem_mini_v2-basic_usb_autoboot_blob_jail-Release-27.1.rom`` file onto your USB drive, then eject the drive and insert it into your (powered down) Librem Mini.  USB 3.0 (blue ports) are faster.
-
-    .. note:: You'll need a monitor and keyboard plugged into your Librem Mini for this operation
+    .. caution:: USB drive **must** be formatted FAT32
```
Comment (gStart9) [inline site/source/guides/flashing/flashing-firmware.rst:10]:
```suggestion
    .. caution:: The USB drive **must** be formatted with a FAT32 filesystem
```

File: site/source/guides/flashing/flashing-firmware.rst [hunk 3]
```diff
@@ -3,24 +3,19 @@
...
 
-#. Turn on the Librem Mini while pressing the ``ESC`` key on the keyboard repeatedly while you see this screen:
+#. `Download the latest firmware from Purism's git repository <https://source.puri.sm/firmware/releases/-/blob/master/librem_mini_v2/custom/pureboot-librem_mini_v2-basic_usb_autoboot_blob_jail-Release-28.rom.gz>`_ and extract it.
```
Comment (gStart9) [inline site/source/guides/flashing/flashing-firmware.rst:12]:
```suggestion
#. `Download the latest firmware from Purism's git repository <https://source.puri.sm/firmware/releases/-/raw/master/librem_mini_v2/custom/pureboot-librem_mini_v2-basic_usb_autoboot_blob_jail-Release-28.rom.gz>`_ and extract it.
```

Comment (gStart9) [reply site/source/guides/flashing/flashing-firmware.rst:12]: Direct link to file is better IMO: more direct, less confusing

File: site/source/guides/device-guides/dg-mac/backup-mac.rst [hunk 4]
```diff
@@ -61,10 +61,6 @@ Please select what version of MacOS you are using from the two tabs below:
 
              .. tip:: You can find hostname at the bottom of sharing window.  You will need this in the next step.
```
Comment (gStart9) [inline site/source/guides/device-guides/dg-mac/backup-mac.rst:62]:
```suggestion
             .. tip:: You can find the hostname at the bottom of sharing window.  You will need this in the next step.
```

**Review by gStart9 — CHANGES_REQUESTED** [review]

File: site/source/guides/flashing/flashing-firmware.rst (hunk 3 above)
Comment (kn0wmad) [reply site/source/guides/flashing/flashing-firmware.rst:12]: Agreed

**Review by gStart9 — APPROVED** [review]

**Pull request merged**